### PR TITLE
[TTS] [bugfix] German FastPitch HiFi-GAN tutorial

### PR DESCRIPTION
**Pull request by aroraakshit**
# What does this PR do ?

Address QA bugs for German TTS tutorial

**Collection**: TTS

# Changelog 
- tutorial notebook changed

**PR Type**: Bugfix

### CODE REVIEW

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb [hunk 1]
```diff
@@ -0,0 +1,1226 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b8813aef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# lets download the files we need to run this tutorial\n",
+    "\n",
+    "!mkdir NeMoGermanTTS\n",
+    "!cd /NeMoGermanTTS && wget https://raw.githubusercontent.com/nvidia/NeMo/$BRANCH/scripts/dataset_processing/tts/openslr/get_data.py\n",
```
Comment (rlangman) [inline tutorials/tts/FastPitch_Training_GermanTTS.ipynb:88]: Would be nice if repeated values like "NeMoGermanTTS" could be changed to a constant/variable

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb [hunk 2]
```diff
@@ -0,0 +1,1226 @@
...
+   "id": "3ae1c439",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from phonemizer.backend import EspeakBackend\n",
+    "import json\n",
+    "\n",
+    "backend = EspeakBackend('de')\n",
+    "\n",
+    "input_manifest_filepaths = [\"DataGermanTTS/thorsten-de/train_manifest\", \"DataGermanTTS/thorsten-de/test_manifest\", \"DataGermanTTS/thorsten-de/val_manifest\"]\n",
```
Comment (rlangman) [inline tutorials/tts/FastPitch_Training_GermanTTS.ipynb:375]: Line is too long to see in notebook editor

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb [hunk 3]
```diff
@@ -0,0 +1,1226 @@
...
+   "id": "990761d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!(cd NeMoGermanTTS && \\\n",
+    "  python hifigan_finetune.py --config-path NeMoGermanTTS --config-name hifigan.yaml \\\n",
+    "    model.max_steps=10 \\\n",
+    "    model.optim.lr=0.00001 \\\n",
+    "    model.train_ds=train_ds_finetune \\\n",
+    "    model.validation_ds=val_ds_finetune \\\n",
```
Comment (rlangman) [inline tutorials/tts/FastPitch_Training_GermanTTS.ipynb:1122]: I get the below error unless I change the "." to "/" in the arguments. Is the behavior here inconsistent?

```
...
    raise ValueError(f"No dataset for {name}")
ValueError: No dataset for train
```

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb [hunk 4]
```diff
@@ -0,0 +1,1226 @@
...
+    "\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from nemo.collections.tts.torch.helpers import BetaBinomialInterpolator\n",
+    "\n",
+    "folder_name = \"synmels\"\n",
+    "fastpitch_model_path = \"<path_to_fastpitch_nemo_or_ckpt>\"\n",
+    "dataset_part_full = \"test_manifest_phonemes\" # or \"val_manifest_phonemes\", \"train_manifest_phonemes\"\n",
+    "dataset_part = \"test_phonemes\" # or \"val_phonemes\", \"train_phonemes\"\n",
+    "dataset_base_path = \"DataGermanTTS/\"\n",
```
Comment (rlangman) [inline tutorials/tts/FastPitch_Training_GermanTTS.ipynb:937]: Is there a reason not to loop over the 3 necessary parts? Like:

```
for part in ["train", "test", val"]:
    dataset_part = f"{part}_phonemes"
    dataset_part_full = f"{part}_manifest_phonemes"
    ....
```

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb [hunk 5]
```diff
@@ -0,0 +1,1226 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea82b646",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def evaluate_spec_fastpitch_ckpt(spec_gen_model, v_model, test):\n",
+    "    with torch.no_grad():\n",
+    "        torch.manual_seed(seed)\n",
+    "        torch.cuda.manual_seed(seed)\n",
```
Comment (rlangman) [inline tutorials/tts/FastPitch_Training_GermanTTS.ipynb:670]: Isn't FastPitch inference deterministic? Does it need a seed?

**Review by redoctopus — DISMISSED** [review]
LGTM.

Just for the future, please use `git mv` to avoid large diffs.

File: tutorials/tts/FastPitch_Training_GermanTTS.ipynb (hunk 3 above)
Comment (aroraakshit) [reply tutorials/tts/FastPitch_Training_GermanTTS.ipynb:1122]: corrected

**Comment by aroraakshit** [comment]
correcting signed commits in PR: https://github.com/NVIDIA/NeMo/pull/4459 instead of this.

**Pull request closed without merging**